src/global.cpp

```diff
@@ -926,11 +925,24 @@ void fa_write (int j, str outdir, const MTL & M, const Plates & P, const FP & pp
                 } else {
                     //we aren't supposed to know the kind  use priority instead
                     //strncpy(objtype[i], F.kind[G[g].id].c_str(), objtypelen);
+                    fibermask[i] = 0;
                     ra[i] = M[g].ra;
                     dec[i] = M[g].dec;
                     dpair proj = projection(g,j,M,P);
                     x_focal[i] = proj.f;
                     y_focal[i] = proj.s;
+                    
+                    //testing that xy2radec is working with good precision.
+                    xy2radec(&testra, &testdec, tilera, tiledec, x_focal[i], y_focal[i]);
+                    if(fabs(testra-ra[i])>(1.0/3600.0)){//arcsecond precision
```

Comment:
Please check at a precision of 0.01 arcsec instead of 1 arcsec for both ra and dec.  In the RA direction, please include the cos(dec_radians) term so that you are testing a real separation on the sky and not just delta(ra).

Reply:
That implementation was producing offsets on the order of 0.5 arcseconds. I had to rewrite the implementation for radec2xy in order to have consistent transformations radec2xy<->xy2radec at machine precision.